desktop-app/app/reducers/browser.js

```diff
@@ -259,14 +262,16 @@ export default function browser(
     },
     isInspecting: false,
     windowSize: getWindowSize(),
+    existingSearchResults: getExistingSearchResults()
```

Comment:
I don't think we need this in the reducer, it can just be a field in `app/services/url-suggestion/index.js` module and it can be internal to that module. 
This doesn't really affect the UI, so this is just an unnecessary addition to the state if we keep it here.